dynamo/dynamo.go

```diff
@@ -15,12 +15,23 @@ type Event struct {
 
 // Record represents a single Dynamo record.
 type Record struct {
-	EventID      string                        `json:"eventID"`
-	EventName    string                        `json:"eventName"`
-	EventSource  string                        `json:"eventSource"`
-	EventVersion string                        `json:"eventVersion"`
-	AWSRegion    string                        `json:"awsRegion"`
-	Dynamodb     *dynamodbstreams.StreamRecord `json:"dynamodb"`
+	EventID      string        `json:"eventID"`
+	EventName    string        `json:"eventName"`
+	EventSource  string        `json:"eventSource"`
+	EventVersion string        `json:"eventVersion"`
+	AWSRegion    string        `json:"awsRegion"`
+	Dynamodb     *StreamRecord `json:"dynamodb"`
+}
+
+// StreamRecord represents a Dynamo stream records
+type StreamRecord struct {
+	ApproximateCreationDateTime int64
+	Keys                        map[string]*dynamodb.AttributeValue
+	NewImage                    map[string]*dynamodb.AttributeValue
+	OldImage                    map[string]*dynamodb.AttributeValue
+	SequenceNumber              string
+	SizeBytes                   int64
+	StreamViewType              string
```

Comment:
No, tags are not required since the JSON keys have the same casing.

Example:
```
{
  "eventID": "a4667189b69049238be6e19dca1b0b68",
  "eventName": "INSERT",
  "eventVersion": "1.1",
  "eventSource": "aws:dynamodb",
  "awsRegion": "us-east-1",
  "dynamodb": {
    "ApproximateCreationDateTime": 1483834200,
    "Keys": {
      "id": {
        "S": "123"
      }
    },
    "NewImage": {
      "id": {
        "S": "123"
      },
      "name": {
        "S": "bob"
      }
    },
    "SequenceNumber": "123456700000000012345678901",
    "SizeBytes": 100,
    "StreamViewType": "NEW_IMAGE"
  },
  "eventSourceARN": "arn:aws:dynamodb:us-east-1:123456789012:table\/table_name\/stream\/2017-01-01T00:00:00.000"
}
```

[docs](http://docs.aws.amazon.com/dynamodbstreams/latest/APIReference/API_StreamRecord.html)